process/interceptors/multiDataInterceptor.go

```diff
@@ -152,28 +146,25 @@ func (mdi *MultiDataInterceptor) ProcessReceivedMessage(message p2p.MessageP2P,
 				"is for this shard", isForCurrentShard,
 				"is white listed", isWhiteListed,
 			)
-			wgProcess.Done()
-			continue
```

Comment:
Return error here as to not allow broadcast of combined shard data.

Reply:
done